test/support/tester/utils.ts

```diff
@@ -114,3 +114,17 @@ export function createSubscription(playerId?: string): Subscription {
   subscription.createdAt = new Date(2017, 11, 13, 2, 3, 4, 0).getTime();
   return subscription;
 }
+
+export function isArrayContentsEqual<T>(arr1:Array<T>, arr2:Array<T>){
```

Comment:
Since this is for tests only for test we can use lodash's `isEqual` which we already have in our package.json.
https://stackoverflow.com/a/29951380/1244574

It's also possible this is in ava but haven't looked.